pkg/prove/proof.go

```diff
@@ -1,19 +1,24 @@
 package prove
 
 import (
+	"bytes"
 	"errors"
 
+	"github.com/celestiaorg/celestia-app/pkg/da"
+
```

Comment:
Extra line

```suggestion

```

Reply:
this is done by `gofumpt` formatter

Reply:
yes, when the ide auto adds an import, `gofumpt` separates the standard imports from the external ones, which is why there are three blocks of imports now

Reply:
fixed in this PR: https://github.com/celestiaorg/celestia-app/pull/1072/files